x-pack/test/functional/apps/transform/creation_index_pattern.ts

```diff
@@ -149,6 +149,7 @@ export default function ({ getService }: FtrProviderContext) {
           },
           row: {
             status: TRANSFORM_STATE.STOPPED,
+            type: 'pivot',
```

Comment:
We don't necessarily need to specify the type in the expected row data again, as we already have it as `testData.type`, so instead of `testData.expected.row.type` we could use `testData.type`.
(this goes for all the test data, I'll not repeat this comment for all of them)

Reply:
Done in https://github.com/elastic/kibana/pull/106990/commits/d407a8a112e3a73e375352cff81f9622c77699d2